textual_fspicker/base_dialog.py

```diff
@@ -70,8 +72,10 @@ class FileSystemPickerScreen(ModalScreen[Optional[Path]]):
     BINDINGS = [Binding("full_stop", "hidden"), Binding("escape", "dismiss(None)")]
     """The bindings for the dialog."""
 
+    select_Changed = False
+
     def __init__(
-        self, location: str | Path = ".", title: str = "", select_button: str = ""
+        self, location: str | Path = ".", title: str = "", select_button: str = "", drives: list[str] = []
```

Comment:
Mutable defaults in method arguments are normally a really bad idea. I don't see why it would be used here.

Reply:
dirt of an old implementation